lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reuse-existing-identity.yaml

```diff
@@ -91,6 +112,36 @@ DETAILS_DELETED_PAGE:
       targetJourney: NEW_P2_IDENTITY
       targetState: START
 
+UPDATE_DETAILS_PAGE:
+  response:
+    type: page
+    pageId: update-details
+  events:
+    address:
+      targetJourney: UPDATE_ADDRESS
+      targetState: START
+    name:
+      # TODO replace with UPDATE_NAME_ADDRESS when it is complete
+      targetJourney: UPDATE_ADDRESS
+      targetState: START
+    name-address:
+      # TODO replace with UPDATE_NAME_ADDRESS when it is complete
+      targetJourney: UPDATE_ADDRESS
+      targetState: START
+    names-dob:
+      targetState: UPDATE_NAME_DOB_PAGE
+    cancel:
+      targetState: RETURN_TO_RP
+
+UPDATE_NAME_DOB_PAGE:
+  response:
+    type: page
+    pageId: update-name-date-birth
+    context: coi
```

Comment:
Does this page need updating to support this context value? Are we missing a ticket...

Reply:
@Joe-Edwards-GDS . Yeh I think so the wording looks slightly different on the ticket and there is an extra hint. I've got a branch with the required changes in, was going to sneak it in with this ticket :)

Reply:
:eyes:

Sounds good, thanks for spotting!